package.json

```diff
@@ -10,6 +10,7 @@
     "assert": "^1.4.1",
     "async": "2.5.0",
     "aws-sdk": "2.141.0",
+    "basic-auth": "1.1.0",
```

Comment:
@sandeepmjay Just curious, why not `basic-auth: 2.0`?  https://github.com/jshttp/basic-auth/releases  Not a merge blocker